plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/LaunchYourStore.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Automattic\WooCommerce\Admin\Features\OnboardingTasks\Tasks;
+
+use Automattic\WooCommerce\Admin\Features\Features;
+use Automattic\WooCommerce\Admin\Features\OnboardingTasks\Task;
+
+/**
+ * Launch Your Store Task
+ */
+class LaunchYourStore extends Task {
+	/**
+	 * ID.
+	 *
+	 * @return string
+	 */
+	public function get_id() {
+		return 'launch_your_store';
+	}
+
+	/**
+	 * Title.
+	 *
+	 * @return string
+	 */
+	public function get_title() {
+		return __( 'Launch your store', 'woocommerce' );
+	}
+
+	/**
+	 * Content.
+	 *
+	 * @return string
+	 */
+	public function get_content() {
+		return __(
+			"It's time to celebrate – you're ready to launch your store! Woo! Hit the button to preview your store and make it public.",
+			'woocommerce'
+		);
+	}
+
+	/**
+	 * Time.
+	 *
+	 * @return string
+	 */
+	public function get_time() {
+		return '';
+	}
+
+	/**
+	 * Task completion.
+	 *
+	 * @return bool
+	 */
+	public function is_complete() {
+		$launch_status = get_option( 'launch-status' );
```

Comment:
Using the same logic as the bridge. `launch-status` option is used in wpcom. We should set the default value to `unlaunched` when we're working on https://github.com/woocommerce/woocommerce/issues/45114.

https://github.com/Automattic/wc-calypso-bridge/blob/master/includes/tasks/class-wc-calypso-task-launch-site.php#L85-L95

Reply:
Sorry for the late reply here. I was just trying to use this option and I'm thinking it may be better as `woocommerce_launch_status`. I'm just looking at the other options we use while adding it to this list. What do you think?

https://github.com/woocommerce/woocommerce/blob/64906823664245e6a37391315129c9103ee912cb/plugins/woocommerce/src/Admin/API/Options.php#L157-L226

Reply:
That sounds good to me. I don't have a strong opinion on the name. 🙂  Let's go with that.

Reply:
Hey @psealock I noticed that you're working on https://github.com/woocommerce/woocommerce/issues/45111. Would it be better to use this [`woocommerce_coming_soon`](https://github.com/woocommerce/woocommerce/issues/45107#issuecomment-1971213585)?

I think launch_status is more like to determine if a store has been launched once or not so we don't show the task again when it was previously completed. I found that wp.com is using `launch-status` and `wpcom_public_coming_soon` flags too.